.github/workflows/phpstan.yml

```diff
@@ -0,0 +1,17 @@
+name: phpstan
+
+on: [push, pull_request]
+
+jobs:
+  phpcs:
+      name: phpstan
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout code
+          uses: actions/checkout@v2
+
+        - name: Composer update
+          run: composer install --dev --prefer-dist --no-progress --no-suggest
```

Comment:
You are using the deprecated option "dev". Dev packages are installed by default now.